webapp/sockets.py

```diff
@@ -131,8 +131,9 @@ def build_wapypoints(waypoints, clear):
 def handle_gps_request():
     print('gps data sent')
     NESW = (0, 0)
-    if gps is not None:
-        gps.get_data()
+    if len(gps) > 0:
```

Comment:
convention states this line should read `if gps:`

going forward with the comment:

> # TODO: Figure out how to handle multiple GPS readings

I plan to have a GPS muxing class that takes into account the dilutions of precision attributes and differences of latitude and longitudes... Something similar should be devised for aggregating sensor data from multiple IMUs, but that will be a little more complex since each sensor device has a different set of capabilities (gyro + accel vs mag only vs gyro + accel + mag). Opening a issue on this idea.